### DISCUSSION TITLE
Can't get packages to release.

### DISCUSSION THREAD

**Discussion by dreadylocks** [community/community#28535] [Actions]
I've got a project that was bootstrapped with electron react boilerplate so most of my workflows were based on those. I modified them to use the published electron builder action after I was getting some weird errors. Well that fixed the errors but I cannot get my app to publish. 

Here's the repo https://github.com/sstephans/bookvox.

According to the example, in order to release I should make a commit, tag it in the format of vX.X.X and then push both the commit and the tag. But when I do this, everything is successful but I get this in my publish step

```
• skipped publishing  file=bookvox-1.0.0.AppImage reason=release doesn't exist and not created because "publish" is not "always" and build is not on tag tag=v1.0.0 version=1.0.0
  • skipped publishing  file=latest-linux.yml reason=release doesn't exist and not created because "publish" is not "always" and build is not on tag tag=v1.0.0 version=1.0.0
```

I am also seeing that the actual tag push for some reason is only triggering my test workflow and not my publish workflow. Only actual code pushes trigger both.

<img width="646" alt="image" src="https://user-images.githubusercontent.com/29578272/183232799-1877df6a-6efe-4a89-93c2-f7833a6c8150.png">

I've also tried adding `release: always` and `publish: always` to the yaml but I always get the same issue. It does release the zipped source code but it's not releasing the installers that are built. I'm new to gitlab actions and the CI/CD configuration. Can someone take a look at my workflows and see if I'm doing something wrong please?

**Comment by radar-blip** [comment]
Somebody else was having a similar problem yesterday, I dont know if they got it straigtened out or if it was a bug/glitch. 
Possibly a glitch. Anyway, I cant help you with it but I hope you get it figured out! 👍🏼
